src/simple-scrollbar.js

```diff
@@ -37,7 +37,36 @@
       d.removeEventListener('mouseup', stop);
     }
   }
-
+  function getHiddenParent(element) {
+    var hiddenParent = "";
```

Comment:
hidden should not be a string

Reply:
then shall I leave the variable unassigned?